### PR TITLE
Added more detailed output and error handling

### LINKED ISSUE #34: Improve feedback/error messages in cli
Hey,

I have recently started to use your tool. 
However, there were a few things that bothered me.
If you do not mind, I would like to contribute and improve feed back messages in the CLI and various other stuff. <b>Feel free to assign me to this issue</b>.
Just posting this issue so noone else works on the same stuff at the same time 😄.

### PR DESCRIPTION
**Pull request by zaanposni**
- Added more feedback/output
- Using custom error handling (this can be extended even more)
- Using `python -m pip` instead of `pip`
- Also I tried to respect the line limit for pep8. However, we can remove that commit if you do not want to do so. At least I am not a fan of the pep 8 line limit.

Tested with:
```
Python2.7.17
Python3.6.9
Python3.7.5
Ubuntu 18.04.4 LTS
```

Note:
I am not familiar with docker, so I did not know if I have to edit something in the Dockerfile as well.

fixes #34

### CODE REVIEW

File: cloudflare_ddns_src/customerrors.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env python
+
+
+class FetchExternalIPError(Exception):
```
Comment (LINKIWI) [inline cloudflare_ddns_src/customerrors.py]: This doesn't seem very necessary to me. Why do we need a dedicated exception class?

This also introduces additional overhead needing to install and include an addditional directory `cloudflare_ddns_src`; I'd rather everything be contained in a single file.

Comment (zaanposni) [reply cloudflare_ddns_src/customerrors.py]: I used the custom exceptions to handle errors more easily at [this line](https://github.com/LINKIWI/cloudflare-ddns-client/pull/35/files#diff-313c56cbe7ec120d363492f37306ec95R288). Also see [here](https://stackoverflow.com/a/417457/9850709).

However, I wasn't sure about introducing new files either. 
It provides better overview of your souce code and it should be safe installed by your makefile.
On the other hand, I also see the benefits of one-file projects.

My aim was to provide detailed information, so noone has to check the source code to understand what is failing. But I will change it as you want to :)

**Comment by zaanposni** [comment]
fixed @LINKIWI

**Comment by LINKIWI** [comment]
Looks good to me. Thanks!

**Pull request merged**